pom.xml

```diff
@@ -46,6 +76,23 @@
           </descriptorRefs>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.3</version>
+        <configuration>
+          <compilerArgument>
+            -Xlint:unchecked,deprecation
+          </compilerArgument>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
No strong opinion here. I just saw a build warning from maven, and googled up a way to make it quiet :)

I believe this just adds a few more standard maven dependencies to pull and run at some point of release cycle, so should not be fatal except on isolated build hosts.